extensions/adobe/experience/offer-management/content/html-content.schema.json

```diff
@@ -0,0 +1,38 @@
+{
+  "meta:license": [
+    "Copyright 2017 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "$id":
+    "https://ns.adobe.com/experience/offer-management/content/html-content",
+  "type": "object",
+  "title": "Content",
+  "description": "Offer HTML Content",
```

Comment:
This should extend the `content` schema

Reply:
@jbeckert please comment

Reply:
In the long run we want to do that. For now our content is not an entity that is managed in a repository as just a self-standing, independently existing object. By extending `https://ns.adobe.com/xdm/content/content` we would implicate an `@id`, `repo:assetId` and various other `repo:` properties. Btw the conflict between `@id` and `repo:assetId` is still pending a resolution.

We also don't support the Content Structured Resource semantic such as versioning, blob/metadata separation, renditions etc for our simple and embedded content. Deriving from the said content schema would document something that isn't true.

We need to address this at the appropriate time. The path to do that will be by extending the base for our content: https://ns.adobe.com/experience/offer-management/representation-content